modules/microshift-firewall-opt-settings.adoc

```diff
@@ -70,3 +70,4 @@ $ sudo firewall-cmd --permanent --zone=public --add-port=80/tcp
 $ sudo firewall-cmd --permanent --zone=public --add-port=443/tcp
```

Comment:
@pliurh are you referring to this section about the ports? And if so, do you know of a better example command we could use for the procedure, "Configuring ports for applications exposed through the router:"

I could also fix this in a z-stream, but do the other updates in this PR look good?

Reply:
Yes. There is no need to "Configuring ports for applications exposed through the router". Since the router pods use `hostports` which are not subjected to firewalld.

The other update in this PR looks good to me.

Reply:
Okay, thank you! I'm gonna remove those commands from the docs then